doctr/models/recognition/postprocessor.py

```diff
@@ -37,21 +37,18 @@ def ctc_decoder(
 
         """
         batch_len = tf.cast(tf.shape(logits)[0], dtype=tf.int64)
-        sequence_len = tf.cast(tf.shape(logits)[1], dtype=tf.int64)
+        sequence_len = tf.cast(tf.shape(logits)[1], dtype=tf.int32)
 
         # computing prediction with ctc decoder
-        decoded_logits = K.ctc_decode(
-            tf.nn.softmax(logits), sequence_len * tf.ones(shape=(batch_len,), dtype=tf.int64), greedy=True
+        _prediction = tf.nn.ctc_greedy_decoder(
+            tf.nn.softmax(tf.transpose(logits, perm=[1, 0, 2])),
+            sequence_len * tf.ones(shape=(batch_len,), dtype="int32"),
+            merge_repeated=True
         )
-        _predictions = tf.squeeze(decoded_logits[0])
+        _prediction = _prediction[0][0]
+        prediction = tf.sparse.to_dense(_prediction, default_value=self.num_classes)
```

Comment:
CI seems to fail here, I think it's `len(self.vocab) + 1`

Reply:
len(self.vocab) = num_classes so I think len(self.vocab)